launch/launch/main.py

```diff
@@ -22,25 +22,86 @@
 
 
 def file_exists(filename):
-    if not os.path.exists(filename) or not os.path.isfile(filename):
-        raise argparse.ArgumentError("'%s' does not exist" % filename)
-    return filename
+    return os.path.exists(filename) and os.path.isfile(filename)
+
+
+def get_launch_arg(arg, separator=":="):
+    # Make sure the separator exists
+    if separator not in arg:
+        return None
```

Comment:
This check is redundant with the one below. If `:=` is not in the string then a list of length one with be returned. I don't think checking this before hand will really save much time especially since this is a one time cost at startup of a launch file.